scripts/pysa_fuzzer/example.py

```diff
@@ -0,0 +1,445 @@
+
```

Comment:
This is useful to see the output, but we probably don't want to merge this. Can you remove it in the final PR?